MerlinAU.sh

```diff
@@ -2237,73 +2270,120 @@ _Set_FW_UpdatePostponementDays_()
    return 0
 }
 
+##-------------------------------------##
+## Added by Martinski W. [2024-Feb-22] ##
+##-------------------------------------##
+_ValidateCronJobSchedule_()
```

Comment:
I see, so basically every input now is validated against these new regular expressions.
This addition really enhances the robustness of the cron section of the script by preventing incorrect cron schedule entries :) I like it!